mmcv/parallel/data_parallel.py

```diff
@@ -56,6 +57,9 @@ def train_step(self, *inputs, **kwargs):
         if not self.device_ids:
             # We add the following line thus the module could gather and
             # convert data containers as those in GPU inference
+            if hasattr(torch, 'is_mlu_available') and torch.is_mlu_available():
+                inputs, kwargs = self.scatter(inputs, kwargs, [0])
+                return self.module.train_step(*inputs[0], **kwargs[0])
```

Comment:
I do not know the meaning of these lines, is it any differences with passing `device_ids=[0]`? I think we can judge device in scatter.

Reply:
Okay, I'll try and test it now.